source/scripting/Ped.cs

```diff
@@ -1150,6 +1281,30 @@ public void LeaveGroup()
 			Function.Call(Hash.REMOVE_PED_FROM_GROUP, Handle);
 		}
 
+		public void PlayAmbientSpeech(string speechName, SpeechModifier modifier = SpeechModifier.Standard)
+		{
+			if ((int) modifier >= 0 && (int) modifier < _speechModifierNames.Length)
+			{
+				Function.Call(Hash._PLAY_AMBIENT_SPEECH1, Handle, speechName, modifier);
+			}
+			else
+			{
+				new ArgumentOutOfRangeException("modifier");
+			}
+		}
+
+		public void PlayAmbientSpeech(string voiceName, string speechName, SpeechModifier modifier = SpeechModifier.Standard)
+		{
+			if ((int)modifier >= 0 && (int)modifier < _speechModifierNames.Length)
+			{
+				Function.Call(Hash._PLAY_AMBIENT_SPEECH1, Handle, speechName, voiceName, modifier, 0);
```

Comment:
Shouldnt this be _PLAY_AMBIENT_SPEECH_WITH_VOICE, also this goes for this and the above native, Pretty sure that modifier is meant to be called using _speechModifierNames[modifier]